bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/handler/BlueGigaBridgeHandler.java

```diff
@@ -341,7 +341,6 @@ public void bluegigaEventReceived(@Nullable BlueGigaResponse event) {
                 device = new BlueGigaBluetoothDevice(this, new BluetoothAddress(scanEvent.getSender()),
                         scanEvent.getAddressType());
                 devices.put(sender, device);
-                deviceDiscovered(device);
```

Comment:
I think this has to be removed, as device discovery is handled by the Device itself. Otherwise, discovery participants get notified before important information like manufacturerId are added.